tests/schemathesis/hooks.py

```
@@ -0,0 +1,72 @@
+            timeout=10,
+        )
+        resp.raise_for_status()
+        data = resp.json()["data"]
+        _session_token = data["sessionToken"]
+        _agent_id = data["agentId"]
+        _room_id = data["roomId"]
+        print(
+            f"[hooks] Registered agent {_agent_id} in room {_room_id}",
+            file=sys.stderr,
+        )
+    except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)